airflow-core/tests/unit/api_fastapi/core_api/routes/public/test_dags.py

```diff
@@ -507,8 +507,10 @@ class TestDagDetails(TestDagEndpoint):
         ],
     )
     @pytest.mark.usefixtures("configure_git_connection_for_dag_bundle")
+    @mock.patch("airflow.api_fastapi.core_api.datamodels.dag_versions.hasattr")
```

Comment:
Non blocking, but create a fixture here?
(skip if you think it doesn't make sense)

Reply:
Skipping. These are temporary tests that would be removed once provider minimum supported version is 3.1